package.json

```diff
@@ -565,6 +595,12 @@
         "command": "vscode-docker.api.configure",
         "title": "Add Docker files to Workspace (API)"
       },
+      {
+        "command": "vscode-docker.ACR-queueBuild",
+        "title": "Queue Build",
```

Comment:
We need to hide this command if the Azure extension is not installed. Talk to me about some options.

Reply:
I agree with that. I didn't even think about that before, but I'm not sure what you mean by options Stephen. Could you elaborate, please?